packages/core/helper-plugin/src/utils/getAPIInnerErrors.ts

```diff
@@ -0,0 +1,38 @@
+import { AxiosError } from 'axios';
+
+import { normalizeAPIError } from './normalizeAPIError';
+
+import type { ApiError } from '../types';
+import type { MessageDescriptor } from 'react-intl';
+
+interface GetAPIInnerErrorsOptions {
+  getTrad: (id: string) => string;
+}
+
+/**
+ *
+ * Returns a normalized error message
+ *
+ * @deprecated
```

Comment:
```suggestion
 * @deprecated
 * @preserve
```

I don't think this carries over if you don't have `preserve`